Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: run-explorer run-tests run-linters build-ui build-python build-docker run-docker compose-up
 
-version="0.12.0"
+version="0.12.2"
```

Comment:
Expecting to merge after Nikola?

Reply:
Yes but now I see that maybe this PR will go first